views/dm.jade

```diff
@@ -20,6 +20,16 @@ block content
 
         #map-wrapper
         #dm-toolbar.toolbar-wrapper
+            button.scroll-button
```

Comment:
I had to cancel the touch event propagation since it would result in scrolling the page while drawing. I added this button which will only be shown on tablets. When using this button the scroll event is not canceled which allows scrolling huge maps.